### PR TITLE
fix #1168

### LINKED ISSUE #1168: Feature: wordpress_import should generate redirections for tags
Hi guys

After I migrated my site to Nikola I noticed that no redirection were generated for tags. It would be helpful if the wordpress_import feature would generate them. 

For categories, it could also be done since categories of posts are transformed into tags, but there are generally less categories. 

Thanks

### PR DESCRIPTION
**Pull request by ralsina**
Add redirects for wordpress tags

### CODE REVIEW

File: nikola/plugins/command/import_wordpress.py [hunk 1]
```diff
@@ -165,6 +166,12 @@ def show_info_about_mising_module(modulename):
             self.extra_languages)
         self.context['REDIRECTIONS'] = self.configure_redirections(
             self.url_map)
+
+        # Add tag redirects
+        for tag in self.all_tags:
+            tag = utils.slugify(tag.decode('utf8'))
+            self.url_map['{}tag/{}'.format(self.context['SITE_URL'], tag)] = self.context['SITE_URL'] + 'categories/' + tag + '.html'
```
Comment (Kwpolska) [inline nikola/plugins/command/import_wordpress.py]: is this guaranteed to be the path? I don’t think so, as it’s either (a) fully customizable or (b) htaccess-dependent…

And it did not take me long (a look in my RSS, in fact) to find [a blog](http://zaufanatrzeciastrona.pl/post/tag/plus-bank/) with `/post/tag/` as the path…

Comment (ralsina) [inline nikola/plugins/command/import_wordpress.py]: But the path is nowhere in the dump that I can see. Worst case, the user can search/replace.

Comment (Kwpolska) [inline nikola/plugins/command/import_wordpress.py]: okay, let’s merge it this way.

Comment (Kwpolska) [inline nikola/plugins/command/import_wordpress.py]: …or let’s not.  you seem to ignore `PRETTY_URLS` here.

Comment (ralsina) [inline nikola/plugins/command/import_wordpress.py]: Seems like the importer never even looks at the existing site's config so far.

I'll see what I can do.

**Comment by ralsina** [comment]
@Kwpolska I made it do "the right thing" which is to use site.config. However, it will not *currently* do the right thing until I fix #1823 

I propose we merge this as is.

**Comment by Kwpolska** [comment]
:+1: for merging, please handle this yourself (in a terminal, unfortunately).

**Pull request merged**